Makefile

```diff
@@ -127,6 +127,10 @@ endif
 ENABLE_EXT_C ?= 1
 $(call set-feature, EXT_C)
 
+# RV32E Base Integer Instruction Set
+ENABLE_EXT_E ?= 0
```

Comment:
Rename to `ENABLE_RV32E`, `ENABLE_BASE_E` or `ENABLE_E`, and consider adding another option for `RV32I` to clarify the base integer instruction set.